add README.md

jgenove1 opened the pull request:
Added README.md to give new contributors a brief guide to initial setup of the project for testing
resolves issue #1 "Add a README for initial setup"

This guide is intended to be a brief introduction to the basic setup of the project. Some basic familiarity with bash and Linux is assumed.

jerreg committed:
add README.md

Added README.md to give new contributors a brief guide to initial setup of the project for testing
resolves issue #1 "Add a README for initial setup"

khammerm commented:
You could add documentation for if somebody wants their website to be externally visible. This would help if someone was on the EC2 Machine.

jerreg committed:
add instructions for remote development

Added instructions for forwarding ports to the local machine when working on a remote server instance. This
allows the developer to test the webpage locally even thought the flask instance is running on a remote server

fixes issue #1 reviewer comments
